Calibration/HcalAlCaRecoProducers/plugins/AlCaIsolatedBunchSelector.cc

```diff
@@ -145,7 +143,7 @@ void AlCaIsolatedBunchSelector::fillDescriptions(edm::ConfigurationDescriptions&
   edm::ParameterSetDescription desc;
   desc.add<edm::InputTag>("triggerResultLabel", edm::InputTag("TriggerResults", "", "HLT"));
   desc.add<std::string>("processName", "HLT");
-  desc.add<std::string>("triggerName", "HLT_HcalIsolatedBunch");
+  desc.add<std::string>("triggerName", "HLT_ZeroBias_IsolatedBunches");
```

Comment:
maybe this trigger bit could be dropped here and use instead the alcarecotriggerbits tag in here: https://github.com/cms-sw/cmssw/blob/master/Calibration/HcalAlCaRecoProducers/python/ALCARECOHcalCalIsolatedBunchSelector_cff.py#L8

Reply:
I am now using only the trigger bits in the new commit